styleguide/derek/examples/solutions.ejs

```diff
@@ -98,6 +98,11 @@
 
 <%- partial('../_partials/solutions/blog-feed.ejs') %>
 
+<!-- Compare Tables -->
+<div class="container standard-padding-full">
+  <%- partial('../_partials/solutions/compare-table.ejs') %>
+</div>
+
```

Comment:
Since this isn't a pattern it shouldn't really go within solutions. Can you please add to the tables.ejs file instead?